security.php

```diff
@@ -202,18 +216,45 @@ function wpcom_vip_username_is_limited( $username, $cache_group ) {
 	 */
 	$ip_threshold = apply_filters( 'wpcom_vip_ip_login_threshold', 50, $ip );
 
-	$ip_username_count = wp_cache_get( $ip_username_cache_key, $cache_group );
-	$ip_count = wp_cache_get( $ip_cache_key, $cache_group );
+	/**
+	 * Login Limiting Username Threshold
+	 * 
+	 * @param string $username Username of the login request
+	 */
+	$username_threshold = 5 * $ip_username_threshold; // Default to 5 times the IP + username threshold
+	$username_threshold = apply_filters( 'wpcom_vip_username_login_threshold', $username_threshold, $username );
 
-	$is_restricted_username = wpcom_vip_is_restricted_username( $username );
+	/**
+	 * Change the thresholds for Password Resets
+	 */
 	if ( 'lost_password_limit' === $cache_group ) {
-		$ip_username_threshold = 3;
-		$ip_threshold = 3;
+		/**
+		 * Password Reset Limiting IP Username Threshold
+		 *
+		 * @param string $ip IP address of the password reset request
+		 * @param string $username Username of the password reset request
+		 */
+		$ip_username_threshold = apply_filters( 'wpcom_vip_ip_username_password_reset_threshold', 3, $ip, $username );
+
+		/**
+		 * Password Reset IP Threshold
+		 *
+		 * @param string $ip IP address of the password reset request
+		 */
+		$ip_threshold = apply_filters( 'wpcom_vip_ip_password_reset_threshold', 30, $ip );
```

Comment:
Are we changing the default value from `3` to `30`?

Reply:
Good catch! I may have changed this to match the 10x difference between `$ip_username_threshold` and `$ip_threshold` when rate-limiting login attempts [here](https://github.com/Automattic/vip-go-mu-plugins/blob/master/security.php#L196-L203).

Since I can't find where I discussed this with anyone else, I'm happy to change this back to `3` since I don't think it has artificially hindered anyone yet and we are providing a filter to change it.